localstripe/resources.py

```diff
@@ -2878,3 +2878,67 @@ def __init__(self, card=None, customer=None, **kwargs):
 
         self.type = 'card'
         self.card = card_obj
+
+
+class Balance(StripeObject):
+    object = 'balance'
+    # Balance does not have id prefix
+    _id_prefix = 'balance_'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        _currencies = ["usd", "eur"]
```

Comment:
You should use only `"` or only `'` not a mix of `'` and `"`.
In localstripe we use `'` as much as possible.
Can you change all `"` into `'`?